internal/connectors/engine/workflow/context.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func infiniteRetryContext(ctx workflow.Context) workflow.Context {
 	return workflow.WithActivityOptions(ctx, workflow.ActivityOptions{
-		StartToCloseTimeout: 60 * time.Second,
+		StartToCloseTimeout: 5 * time.Minute,
```

Comment:
Ideally I'd put that in a constant so you can do the math in the connector (so in the future we don't have to remember those 2 values are linked)
We might want a different timeout per workflow / connector, maybe we can take inspiration of Paul's PR (use a different retryContext for that)

Reply:
Added a custom context just for the FetchNext activities.